templates/serve-stable-diffusion/README.ipynb

```diff
@@ -35,11 +35,12 @@
    "metadata": {},
    "source": [
     "### Step 2: Run the model locally\n",
-    "- Open a new terminal (ctrl+shift+` in VSCode.)\n",
-    "- Run the command below to deploy your model at http://localhost:8000.  \n",
+    "- Run the command below in a VSCode terminal.\n",
```

Comment:
Curious about why we remove the keyboard shortcut?

Reply:
oh I removed it because it's not actually needed, there's already a terminal window open that you can use by default

Reply:
I think we are not gonna open the terminal by default after we switch to ipynb instead of readme.. We may need to add that back

Reply:
oh got it, will re-test and update once that happens